api-platform/composer.json

```diff
@@ -71,7 +76,14 @@
         }
     },
     "require-dev": {
+        "dama/doctrine-test-bundle": "^7.1",
         "doctrine/doctrine-fixtures-bundle": "^3.4",
+        "justinrainbow/json-schema": "^5.2",
+        "phpunit/phpunit": "^9.5",
+        "symfony/browser-kit": "6.2.*",
+        "symfony/css-selector": "6.2.*",
+        "symfony/http-client": "6.2.*",
+        "symfony/phpunit-bridge": "^6.2",
```

Comment:
do we need this?

Reply:
These are required (and recommended by the current doc) via `symfony/test-pack`. We can maybe skip some ( the bridge for instance), but I think it might be best to keep them to facilitate test writing for guides, wdyt ?